script/utils/sim-sequence.sh

```diff
@@ -0,0 +1,308 @@
+#!/bin/bash
+
+
+LOGFILE="/tmp/sim-sequence.log"
```

Comment:
Same comment on `mktmp`, applies to a few places in this file

Reply:
applied in commit 05ac656de8969a7cdae7c05b879592f0f8da4914

Reply:
Whoops forgot to remove this comment correct!